config/nvblox/specializations/nvblox_zed.yaml

```diff
@@ -1,9 +1,6 @@
 /**:
   ros__parameters:
     # esdf settings
-    esdf_slice_height: -0.3
-    esdf_2d_min_height: 0.0
```

Comment:
why are these lines deleted?

Reply:
They are duplicated in this file and in zed_common, so I kept the ones in zed common

Reply:
nvblox_base.yaml*